doc/nrf/protocols/lte/index.rst

```diff
@@ -0,0 +1,217 @@
+It offers higher bandwidth and lower latency compared to other similar technologies.
+This makes it suited for applications such as asset tracking, health monitors, alarm panels, environmental monitoring, and smart parking.
+
+NB-IoT, or Narrowband IoT, is another LPWAN communication standard that was introduced in 3GPP release 13.
+It is designed for indoor coverage, low cost, and long battery life.
+NB-IoT operates in a small radio frequency band of 200 kHz.
+It is ideal for applications like smart metering, environmental monitoring, and smart lighting.
+
+nRF91 Series devices such as nRF9161 DK, nRF9160 DK, and Thingy:91 support both LTE-M and NB-IoT technologies.
+
+Comparison between LTE-M and NB-IoT
+===================================
+
+The following table shows a comparison between LTE-M and NB-IoT in terms of advantages in using any of these standards:
+
++------------------+--------------------------------+--------------------------------+
+|                  | LTE-M                          | NB-IoT                         |
++==================+================================+================================+
+|                  | Higher throughput              | Longer range                   |
++                  +--------------------------------+--------------------------------+
+|                  | Lower latency                  | Better penetration             |
++                  +--------------------------------+--------------------------------+
+|   Strengths      | Better roaming agreements      | Power efficient at lower data  |
+|                  |                                | rate                           |
++                  +--------------------------------+--------------------------------+
+|                  | Power efficient at medium data |                                |
+|                  | rate                           |                                |
++                  +--------------------------------+--------------------------------+
+|                  | Suitable for TCP/TLS secure    |                                |
+|                  | connection                     |                                |
++------------------+--------------------------------+--------------------------------+
+
+You can find the key feature comparison between LTE-M and NB-IoT in the `nWP044 - Best practices for cellular IoT development`_ documentation.
+
+Factors to be considered for LTE
+********************************
+
+Following are some key aspects to ensure LTE support for your SiP using Nordic Semiconductor's devices, such as the nRF91 Series.
+
+LTE bands
+=========
+
+There are 56 LTE bands from 400 MHz to 6 GHz.
+Some bands use :term:`Frequency Division Duplex (FDD)` and some use :term:`Time Division Duplex (TDD)`.
+The different bands are licensed by different Mobile Network Operators (MNO) around the world.
```

Comment:
Should probably say here that there are no known deployments for other than FDD for M1/NB